src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/debian/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -23,7 +23,8 @@
     <ArchitectureSpecificNativeFile Include="$(BinDir)mscorlib.ni.dll"/>
     <ArchitectureSpecificLibFile Include="$(BinDir)mscorlib.dll"/>
     <ArchitectureSpecificToolFile Include="$(BinDir)crossgen"/>
-
+    <ArchitectureSpecificToolFile Include="$(BinDir)libclrjit.so"/>
```

Comment:
Unpublished standalone with `dotnet run` needs mscorlib.ni packaging to run out of the additional probe path. Same reasoning here for the jit.